tests/flytekit/unit/clients/test_raw.py

```diff
@@ -195,6 +225,15 @@ def test_basic_strings(mocked_method):
 
 @patch.object(RawSynchronousFlyteClient, "_refresh_credentials_from_command")
 def test_refresh_command(mocked_method):
-    cc = RawSynchronousFlyteClient(PlatformConfig(auth_mode=AuthType.EXTERNAL_PROCESS))
+    cc = RawSynchronousFlyteClient(PlatformConfig(auth_mode=AuthType.EXTERNALCOMMAND))
+    cc.refresh_credentials()
+    assert mocked_method.called
+
+
+@patch("flytekit.configuration.internal.Credentials.AUTH_MODE.read")
+@patch.object(RawSynchronousFlyteClient, "_refresh_credentials_from_command")
+def test_refresh_from_environment_variable(mocked_method, mock_read_auth_mode_from_env):
+    cc = RawSynchronousFlyteClient(PlatformConfig(auth_mode=None))
```

Comment:
Can you use `PlatformConfig.auto` and mock the relevant env vars using https://docs.pytest.org/en/6.2.x/reference.html#pytest.MonkeyPatch.setenv ?